apps/smart-contracts/external/yield/test/utils/helpers.js

```diff
@@ -356,4 +356,46 @@ module.exports = {
     const totalSupply = await stakingContract.totalSupply()
     return stakingContractBalance.sub(totalSupply)
   },
+
+  /**
+   * For test accuracy, we want to verify the exact reward amount owed to
+   * a user. We cannot rely on `earned()` since it is a view function and
+   * does not take into account the block timestamp when rewards are claimed.
+   *
+   * This function performs the exact calculations that `earned()` does given
+   * a specific timestamp.
+   */
+  async getRewardPerToken(stakingContract, timeRewardsClaimed) {
+    const rewardPerTokenStored = await stakingContract.rewardPerTokenStored()
+    const totalSupply = await stakingContract.totalSupply()
+    if (totalSupply.eq(web3.utils.toBN(0))) return rewardPerTokenStored
+    /**
+     * Convert to BN since timestamp will be a number and it is easier
+     * to keep everything in BN since all on-chain values are returned as BN.
+     *
+     * Unlike ethers BigNumber, web3 BN's arithmetic operations do not
+     * automatically convert to BN, so need to explicitly convert to BN.
+     */
+    const timeRewardsClaimedBN = web3.utils.toBN(timeRewardsClaimed)
+    const periodFinish = await stakingContract.periodFinish()
+    const lastTimeRewardApplicable = timeRewardsClaimedBN.gt(periodFinish)
+      ? periodFinish
+      : timeRewardsClaimedBN
```

Comment:
```suggestion
    const lastTimeRewardApplicable = timeRewardsClaimedBN.lt(periodFinish)
      ? timeRewardsClaimedBN
      : periodFinish
```
keep consistent with the contract's logic (and should've been >= when it was flipped around anyway)